### ISSUE TITLE
Repo's maintainers information

### ISSUE THREAD

**Issue by vhmcck** [swcarpentry/git-novice-es#97]
### What is the problem?

Hi,

I am reaching out on behalf of the SWC LPGC Committee to report a mismatch between the SWC website and your README file. Would it be possible to please update the maintainers' names on the README file for this repository? We understand the information on the SWC website is up-to-date.

Thanks a lot,
Valentina

### Location of problem (optional)

README file

**Comment by clarallebot** [comment]
Hi @vhmcck thank you for the heads up. I created a pull request. Do you have any suggestions on how to document the link from each maintainer's name to the carpentries website? Right now we have links that are supposed to go to a personal page in the carpentries website, but it looks like they all go to a generic instructors page https://carpentries.org/instructors/

For example, under my name the link points to https://software-carpentry.org/team/#llebot_clara but the link actually goes to https://carpentries.org/instructors/ As far as I could tell the same thing happens in other lessons. Thanks for your help! 

@MGomezN and @j-p-courneya can you check that the information I added about you is correct? Thanks!

**Comment by j-p-courneya** [comment]
That looks good to me except I noticed that my name is not on the carpentries instructors page! That's a separate issue though and shouldn't affect this pr. Thanks!

**Comment by vhmcck** [comment]
Thank you so much for creating the PR, @clarallebot. And for pointing to the issue on the Carpentries Instructors' website, @j-p-courneya.

- Regarding your question about linking personal pages, @tobyhodges might be able to clarify what the issue is. @tobyhodges - it seems that the links associated with the maintainers' names are somehow bypassed and replaced by the instructors' website straight away.
- Can you also please advise on how to correct the information on the Carpentries Instructors' website?

Thanks in advance for your help.

Cheers,
Valentina

**Comment by tobyhodges** [comment]
> * Regarding your question about linking personal pages, @tobyhodges might be able to clarify what the issue is. @tobyhodges - it seems that the links associated with the maintainers' names are somehow bypassed and replaced by the instructors' website straight away.

It looks like these links are based on the old Software Carpentry website, but in recent years we have centralised all of the community member listings (Instructors, Maintainers, Trainers, etc) into the carpentries.org site. The links in the README can be updated to point to https://carpentries.org/maintainers/#github_username, e.g. the URL for @clarallebot is https://carpentries.org/maintainers/#clarallebot, for @vhmcck it is https://carpentries.org/maintainers/#vhmcck

> * Can you also please advise on how to correct the information on the Carpentries Instructors' website?

@j-p-courneya before we can display them on our website, community members need to consent to The Carpentries publishing their profile. You can do this by [logging into our community database, AMY](https://amy.carpentries.org), and updating your preferences there. (A good opportunity to also check whether the other information we have about you is correct.) You can log into AMY using your GitHub account.

**Comment by vhmcck** [comment]
Thanks so much for the clarifications, @tobyhodges.
@clarallebot, @j-p-courneya - has the README file been updated?
Let us know if you have any further questions.
Cheers,
Valentina

**Comment by tobyhodges** [comment]
Circling back to confirm that @j-p-courneya now appears on the page listing Maintainers: https://carpentries.org/maintainers/#j-p-courneya Thanks for updating your AMY profile!

**Comment by vhmcck** [comment]
Thanks, @tobyhodges!
